docs/.vuepress/components/ListCard.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div class="list-new max-w-768px">
+    <template v-for="(item, i) in items">
+      <router-link
+        v-if="item.isInternal"
+        :to="item.link"
+        :aria-label="item.title + ' by ' + item.meta"
+        class="tc-text500 pa-1 block w-100 hide-icon"
+      >
+        <article :class="{ 'has-meta': item.meta }">
+          <h3 class="l7 mb-0 mt-0 tc-text500 art-title" :is="itemTitleTag">
```

Comment:
Can you please add a small comment that this `h3` element is in fact overridden depending on the level passed in? This took me a bit to pick up on.